atlp-rwanda/elite-bn-backend
# ft(multiple-languages):translating responses into multi languages

#### What does this PR do?
[what does this PR do?]
> Our barefoot nomad to support multiple languages  [french, English]
#### Description of Task to be completed
> Installed and configured i18n for translating our response
> Translating our controllers in into multiple language
#### How should this be manually tested?
> Go in browser settings add french and english support languages
> run server `npm run dev`
> run our api docs `http://localhost:5000/api-docs/`
> try on any different route and check the response
#### Any background context you want to provide?
> N/A
#### What are the relevant Trello story?
> https://trello.com/c/9SEpvvOh/19-support-multiple-languages
#### Screenshots (if appropriate)
> N/A
